Branch-SDK/Branch-SDK/BNCAppleSearchAds.h

```diff
@@ -14,10 +14,20 @@ NS_ASSUME_NONNULL_BEGIN
 @interface BNCAppleSearchAds : NSObject
 
 @property (nonatomic, assign, readwrite) BOOL enableAppleSearchAdsCheck;
+@property (nonatomic, assign, readwrite) BOOL ignoreAppleTestData;
 
 + (BNCAppleSearchAds *)sharedInstance;
 
-// checks Apple Search Ads and updates preferences.  This acquires a lock on BNCPreferenceHelper.
+// Default delay and retry configuration.  ~p90
+// typically less than 1s delay, up to 3.5s delay on first app start
+- (void)useBranchConfig;
+
+// Apple suggests a longer delay, however this is detrimental to app launch times
+// typically less than 1s delay, up to 14s delay on first app start
+- (void)useAppleConfig;
```

Comment:
Are we locking method name as it is? 

I personally think this method name can be interpreted in multi context, causing confusion to developers.

Reply:
fixed